xesmf/tests/test_frontend.py

```diff
@@ -46,8 +50,14 @@ def test_as_2d_mesh():
 methods_list = ['bilinear', 'conservative', 'nearest_s2d', 'nearest_d2s']
 
 @pytest.mark.parametrize('method', methods_list)
-def test_build_regridder(method):
-    regridder = xe.Regridder(ds_in, ds_out, method)
+@pytest.mark.parametrize('locstream', [True, False])
+def test_build_regridder(method, locstream):
```

Comment:
I don't think `locstream` can be used with `conservative` method. Does it actually work here?

Reply:
no I hacked it on line 57 :)

Reply:
I would use more explicit expression and avoid overwriting `method` internally:

```python
@pytest.mark.parametrize(
    "locstream,method", [
     (False, method_list), 
     (True, ['bilinear', 'nearest_s2d', 'nearest_d2s'])   # conservative method does not work for LocStream
])
```